app/actions.rb

```diff
@@ -147,13 +147,8 @@ def current_meeting
 
   @votes.each do |_, vote|
```

Comment:
it's receiving the key and the value. '_,' is convention when you are not interested in the first part